autoload/wiki/buffer.vim

```diff
@@ -175,6 +175,9 @@ function! s:init_buffer_mappings() abort " {{{1
   endif
 
   call extend(l:mappings, get(g:, 'wiki_mappings_local', {}))
+  if b:wiki.in_journal
+    call extend(l:mappings, get(g:, 'wiki_mappings_journal_local', {}))
```

Comment:
Could I suggest `local_journal` instead of `journal_local`?

Reply:
Done